python/ray/ml/tests/test_torch_predictor.py

```diff
@@ -53,3 +53,14 @@ def test_predict_feature_columns():
 
     assert len(predictions) == 3
     assert predictions.to_numpy().flatten().tolist() == [4, 8, 12]
+
+
+def test_predict_no_preprocessor():
+    checkpoint = Checkpoint.from_dict({MODEL_KEY: model})
+    predictor = TorchPredictor.from_checkpoint(checkpoint)
+
+    data_batch = np.array([[1], [2], [3]])
+    predictions = predictor.predict(data_batch)
+
+    assert len(predictions) == 3
+    assert predictions.to_numpy().flatten().tolist() == [2, 4, 6]
```

Comment:
shouldn't this be
```suggestion
    assert predictions.to_numpy().flatten().tolist() == [1, 2, 3]
```

Reply:
The model still does x2. 

In other examples, it was x4 (model does x2, preprocessor does x2).

Reply:
ah, gotcha!